### PR TITLE
Remove Engine failover, reenable failover tests

### PR DESCRIPTION
**Pull request by alexandraorth**

### CODE REVIEW

File: grakn-engine/src/main/java/ai/grakn/engine/tasks/storage/TaskStateZookeeperStore.java [hunk 1]
```diff
@@ -101,8 +101,11 @@ public TaskId newState(TaskState task){
     @Override
     public Boolean updateState(TaskState task){
         try {
-            // Get the previously stored task
-            TaskState previousTask = (TaskState) deserialize(zookeeper.connection().getData().forPath(taskPath(task)));
+                // Get the previously stored task
+                TaskState previousTask = (TaskState) deserialize(zookeeper.connection().getData().forPath(taskPath(task)));
+
+                // Remove any configuration information from the task state
+                task.clearConfiguration();
```
Comment (aelred) [inline grakn-engine/src/main/java/ai/grakn/engine/tasks/storage/TaskStateZookeeperStore.java]: This is indented 8 spaces, not 4.

File: grakn-test/src/test/java/ai/grakn/generator/TaskStates.java [hunk 2]
```diff
@@ -61,13 +61,7 @@ public TaskStates() {
     public TaskState generate(SourceOfRandomness random, GenerationStatus status) {
         Class<? extends BackgroundTask> taskClass = random.choose(classes);
 
-        TaskId taskId;
-
-        if (newTask) {
-            taskId = TaskId.generate();
-        } else {
-            taskId = TaskId.of(random.choose(ImmutableSet.of("A", "B", "C")));
-        }
```
Comment (aelred) [inline grakn-test/src/test/java/ai/grakn/generator/TaskStates.java]: If we're removing this, we don't need the `@NewTask` annotation anymore.

**Comment by alexandraorth** [comment]
please retest this please - failed with quickcheck null

**Comment by alexandraorth** [comment]
please retest this please - failed with `ai.grakn.test.graql.reasoner.inference.GenealogyTest.testGrandParentship`

**Pull request merged**